tools/rstlint.py

```diff
@@ -20,26 +20,110 @@
-    'parsed-literal', 'pull-quote', 'raw', 'replace',
-    'restructuredtext-test-directive', 'role', 'rubric', 'sectnum', 'sidebar',
-    'table', 'target-notes', 'tip', 'title', 'topic', 'unicode', 'warning',
+    'admonition',
+    'attention',
+    'caution',
+    'class',
+    'compound',
+    'container',
+    'contents',
+    'csv-table',
+    'danger',
+    'date',
+    'default-role',
+    'epigraph',
+    'error',
+    'figure',
+    'footer',
+    'header',
+    'highlights',
+    'hint',
+    'image',
+    'important',
+    'include',
+    'line-block',
+    'list-table',
+    'meta',
+    'note',
+    'parsed-literal',
+    'pull-quote',
+    'raw',
+    'replace',
+    'restructuredtext-test-directive',
+    'role',
+    'rubric',
+    'sectnum',
+    'sidebar',
+    'table',
+    'target-notes',
+    'tip',
+    'title',
+    'topic',
+    'unicode',
+    'warning',
     # Sphinx and Python docs custom ones
-    'acks', 'attribute', 'autoattribute', 'autoclass', 'autodata',
-    'autoexception', 'autofunction', 'automethod', 'automodule', 'centered',
-    'cfunction', 'class', 'classmethod', 'cmacro', 'cmdoption', 'cmember',
-    'code-block', 'confval', 'cssclass', 'ctype', 'currentmodule', 'cvar',
-    'data', 'decorator', 'decoratormethod', 'deprecated-removed',
-    'deprecated(?!-removed)', 'describe', 'directive', 'doctest', 'envvar',
-    'event', 'exception', 'function', 'glossary', 'highlight', 'highlightlang',
-    'impl-detail', 'index', 'literalinclude', 'method', 'miscnews', 'module',
-    'moduleauthor', 'opcode', 'pdbcommand', 'productionlist',
-    'program', 'role', 'sectionauthor', 'seealso', 'sourcecode', 'staticmethod',
-    'tabularcolumns', 'testcode', 'testoutput', 'testsetup', 'toctree', 'todo',
-    'todolist', 'versionadded', 'versionchanged'
+    'acks',
+    'attribute',
+    'autoattribute',
+    'autoclass',
+    'autodata',
+    'autoexception',
+    'autofunction',
+    'automethod',
+    'automodule',
+    'centered',
+    'cfunction',
+    'class',
+    'classmethod',
+    'cmacro',
+    'cmdoption',
+    'cmember',
+    'code-block',
+    'confval',
+    'cssclass',
+    'ctype',
+    'currentmodule',
+    'cvar',
+    'data',
+    'decorator',
+    'decoratormethod',
+    'deprecated-removed',
+    'deprecated(?!-removed)',
+    'describe',
+    'directive',
+    'doctest',
+    'envvar',
+    'event',
+    'exception',
+    'function',
+    'glossary',
+    'highlight',
+    'highlightlang',
+    'impl-detail',
+    'index',
+    'literalinclude',
+    'method',
+    'miscnews',
+    'module',
+    'moduleauthor',
+    'opcode',
+    'pdbcommand',
+    'productionlist',
+    'program',
+    'role',
+    'sectionauthor',
+    'seealso',
+    'sourcecode',
+    'staticmethod',
+    'tabularcolumns',
+    'testcode',
+    'testoutput',
+    'testsetup',
+    'toctree',
+    'todo',
+    'todolist',
+    'versionadded',
+    'versionchanged',
```

Comment:
I like the original version better.

Reply:
You need to say why for this to be actionable.

Reply:
It doesn’t take two or three screens for a human to read the whole list :)

Reply:
I generally use two different styles depending on the situation:
```python
result = function(with, a, few, args,
                  with, short, names)
```
or
```python
result = a_long_named_function(
    with_a_long_named_argument,
    and_another_long_argument,
    and_maybe_an_expression_like,
    'foo' if condition else 'bar',
)
```
In the first example, putting all args on their own line, would be a waste of vertical space, and (IMHO) it's perfectly readable as is (and as the original list we are commenting was).
Readability is not only determined by the readability of the individual lines, it also depends on how easy it is to navigate vertically among functions and classes.

Reply:
Thanks for the reviews and input. I can see your points on length. So, it looks like it comes down to ease of scanning or extra scrolling.

I'm going to go ahead and merge as in the PR. This is the first time that the file has been touched in 2+ years.